### PR TITLE
bookings controller updated with all CRUD actions and Index & Show pa…

### PR DESCRIPTION
**Pull request by antonio-propato**
…ge created for the app - The booking controller needs updating once the user is created and the command has been commented out for now - Seats in the Seed/Car faker updated as it was not populating the field - also added the prefix in the booking routes

Screenshot 2024-02-27 at 15 23 24

Screenshot 2024-02-27 at 15 23 51

### CODE REVIEW

**Review by nfarinello — APPROVED** [review]
All good to go!

**Comment by gamcdonald123** [comment]
Nice. Fine to merge.

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -1,24 +1,51 @@
...
+  before_action :set_booking, only: [:show, :edit, :update, :destroy]
+
   def index
+    # booking.all to be swapped with current_user.bookings once user is created
     @bookings = Booking.all
+    # @bookings = current_user.bookings
   end
 
   def show
-    @booking = Booking.find(params[:id])
```
Comment (gamcdonald123) [inline app/controllers/bookings_controller.rb:7]: Aren't we having a show booking page? With details of the booking?

**Review by gamcdonald123 — APPROVED** [review]
All look good

File: app/controllers/bookings_controller.rb (hunk 1 above)
Comment (antonio-propato) [reply app/controllers/bookings_controller.rb:7]: Yep, I create the method **set_booking**  and added **before_action :set_booking, only: [:show, :edit, :update, :destroy]** at the top.  

Is that what you meant?

**Comment by soukouna94** [comment]
nice!

**Pull request merged**